docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+- Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
i'd rather stick with '-' 

as this is the first entry for this collection, perhaps it can set a precedent?

I believe that the 1.10 contributions mixed the - and * use